env_support/cmake/custom.cmake

```diff
@@ -4,36 +4,28 @@ option(LV_LVGL_H_INCLUDE_SIMPLE
 
 # Option to define LV_CONF_INCLUDE_SIMPLE, default: ON
 option(LV_CONF_INCLUDE_SIMPLE
-       "Use #include \"lv_conf.h\" instead of #include \"../../lv_conf.h\"" ON)
+       "Simple include of \"lv_conf.h\" and \"lv_drv_conf.h\"" ON)
 
-# Option LV_CONF_PATH, which should be the path for lv_conf.h
-# If set parent path LV_CONF_DIR is added to includes
-if( LV_CONF_PATH )
-    get_filename_component(LV_CONF_DIR ${LV_CONF_PATH} DIRECTORY)
-endif( LV_CONF_PATH )
+# Option to set LV_CONF_PATH, if set parent path LV_CONF_DIR is added to
+# includes
+option(LV_CONF_PATH "Path defined for lv_conf.h")
+get_filename_component(LV_CONF_DIR ${LV_CONF_PATH} DIRECTORY)
 
 # Option to build shared libraries (as opposed to static), default: OFF
 option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 
-# Set sources used for LVGL components
 file(GLOB_RECURSE SOURCES ${LVGL_ROOT_DIR}/src/*.c ${LVGL_ROOT_DIR}/src/*.S)
 file(GLOB_RECURSE EXAMPLE_SOURCES ${LVGL_ROOT_DIR}/examples/*.c)
 file(GLOB_RECURSE DEMO_SOURCES ${LVGL_ROOT_DIR}/demos/*.c)
 file(GLOB_RECURSE THORVG_SOURCES ${LVGL_ROOT_DIR}/src/libs/thorvg/*.cpp ${LVGL_ROOT_DIR}/src/others/vg_lite_tvg/*.cpp)
 
-# Build LVGL library
 add_library(lvgl ${SOURCES})
 add_library(lvgl::lvgl ALIAS lvgl)
 
+
 target_compile_definitions(
   lvgl PUBLIC $<$<BOOL:${LV_LVGL_H_INCLUDE_SIMPLE}>:LV_LVGL_H_INCLUDE_SIMPLE>
-              $<$<BOOL:${LV_CONF_INCLUDE_SIMPLE}>:LV_CONF_INCLUDE_SIMPLE>
-              $<$<COMPILE_LANGUAGE:ASM>:__ASSEMBLY__>)
-
-# Add definition of LV_CONF_PATH only if needed
-if(LV_CONF_PATH)
-  target_compile_definitions(lvgl PUBLIC LV_CONF_PATH=${LV_CONF_PATH})
-endif()
+              $<$<BOOL:${LV_CONF_INCLUDE_SIMPLE}>:LV_CONF_INCLUDE_SIMPLE>)
```

Comment:
I'm not sure of what's the original code does. Is it sure to remove them?
`              $<$<COMPILE_LANGUAGE:ASM>:__ASSEMBLY__>)`

Does the `LV_CONF_PATH` removed intentionally?

```
# Add definition of LV_CONF_PATH only if needed
if(LV_CONF_PATH)
  target_compile_definitions(lvgl PUBLIC LV_CONF_PATH=${LV_CONF_PATH})
endif()
```

Reply:
Added back
```
# Add definition of LV_CONF_PATH only if needed
if(LV_CONF_PATH)
  target_compile_definitions(lvgl PUBLIC LV_CONF_PATH=${LV_CONF_PATH})
endif()
```
and it works. That COMPILE_LANGUAGE etc however is the culprit preventing the cmake file from working.